### PR TITLE
minor update in the new user's sudo permissions ..

### PR DESCRIPTION
**Pull request by dimkatsi91**
New created user belongs to the existing 'wheel' group

### CODE REVIEW

File: archon.sh [hunk 1]
```diff
@@ -12,6 +12,12 @@
 #
 #
 
+
+## ADDITION START
+sudoers_file=/etc/sudoers
```
Comment (cerebrux) [inline archon.sh]: Tip: There is no need to save a value in a variable if it is going to be used once :)

File: archon.sh [hunk 2]
```diff
@@ -218,7 +224,17 @@ function chroot_stage {
...
+	
+	## DELETE START ?
+	## echo "$onomaxristi ALL=(ALL) ALL" >> /etc/sudoers
+	## DELETE END ?
+
+	## ADDITION START ?
+	## Since new user belongs to wheel group now (via useradd command), 
+	## just uncomment the appropriate line  from the /etc/sudoers file
+	sed -i "s/# %wheel ALL=(ALL) ALL/%wheel ALL=(ALL) ALL/" $sudoers_file
+	## ADDITION END ?
```
Comment (cerebrux) [inline archon.sh]: As mentioned before there is no need to use a variable for something used only once. Οπότε εδώ καλύτερα να χρησιμοποιείς απευθείας την /etc/sudoers

**Review by cerebrux — CHANGES_REQUESTED** [review]
Κάνε τις αλλαγές που πρότεινα στα σχόλια

**Comment by dimkatsi91** [comment]
Hi,

Just added this variable cause I thought You would like to add
a function for the /etc/sudoers file for 'sanity check' via 'visudo -f -c sudoers' command,
just after the sed command, since a false sed upon this file is going to lock wheel group users for super-user priviledged commands.

Anyway, I just removed this $sudoers_file variable from my forked repo archon.sh script
in develop branch.

Best Regards,
Dimosthenis Katsimardos

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Review by cerebrux — APPROVED** [review]

**Pull request merged**